.test-infra/jenkins/job_LoadTests_Java.groovy

```diff
@@ -17,118 +17,213 @@
+                        publishToBigQuery   : true,
+                        bigQueryDataset     : 'load_test',
+                        bigQueryTable       : 'java_dataflow_batch_GBK_4',
+                        sourceOptions       : """
+                                            {
+                                              "numRecords": 5000000,
+                                              "keySizeBytes": 10,
+                                              "valueSizeBytes": 90
+                                            }
+                                       """.trim().replaceAll("\\s", ""),
+                        fanout              : 4,
+                        iterations          : 1,
+                        maxNumWorkers       : 16,
+                        numWorkers          : 16,
+                        autoscalingAlgorithm: "NONE"
                 ]
         ],
         [
-                title        : 'GroupByKey load test Spark',
+                title        : 'Load test: fanout 8 times with 2GB 10-byte records total',
                 itClass      : 'org.apache.beam.sdk.loadtests.GroupByKeyLoadTest',
-                runner       : CommonTestProperties.Runner.SPARK,
-                jobProperties: [
-                        sparkMaster      : 'local[4]',
-                        publishToBigQuery: true,
-                        bigQueryDataset  : 'load_test_SMOKE',
-                        bigQueryTable    : 'spark_gbk',
-                        sourceOptions    : '{"numRecords":100000,"splitPointFrequencyRecords":1}',
-                        stepOptions      : '{"outputRecordsPerInputRecord":1,"preservesInputKeyDistribution":true}',
-                        fanout           : 10,
-                        iterations       : 1,
+                runner       : CommonTestProperties.Runner.DATAFLOW,
+                jobProperties     : [
+                        project             : 'apache-beam-testing',
+                        appName             : 'load_tests_Java_Dataflow_Batch_GBK_5',
+                        tempLocation        : 'gs://temp-storage-for-perf-tests/loadtests',
+                        publishToBigQuery   : true,
+                        bigQueryDataset     : 'load_test',
+                        bigQueryTable       : 'java_dataflow_batch_GBK_5',
+                        sourceOptions       : """
+                                            {
+                                              "numRecords": 2500000,
+                                              "keySizeBytes": 10,
+                                              "valueSizeBytes": 90
+                                            }
+                                       """.trim().replaceAll("\\s", ""),
+                        fanout              : 8,
+                        iterations          : 1,
+                        maxNumWorkers       : 16,
+                        numWorkers          : 16,
```

Comment:
Do we want same amount of workers for both 4 fanout and 8 fanout?

Reply:
Yes. The idea is to run fanout 8 with twice as much records as for fanout 4 and see if the execution time is the same. I did some runs and I confirmed this happening on Dataflow. 

Thanks for staying alert, Kasia. :)

Reply:
You mean the other way around? fanout8 with half as much data?

Reply:
Fanout 4 => twice as much, you're right! :)